python/tests/test_highlevel.py

```diff
@@ -2363,12 +2363,19 @@ def test_seek(self):
             self.assertRaises(ValueError, tree.seek, bad_position)
 
     def test_interval(self):
-        ts = msprime.simulate(10)
-        tree = ts.first()
-        self.assertEqual(tree.interval[0], 0)
-        self.assertEqual(tree.interval.left, 0)
-        self.assertEqual(tree.interval[1], 1)
-        self.assertEqual(tree.interval.right, 1)
+        ts = msprime.simulate(10, recombination_rate=1, random_seed=1)
+        self.assertGreater(ts.num_trees, 1)
+        breakpoints = list(ts.breakpoints())
+        self.assertEqual(breakpoints[0], 0)
+        self.assertAlmostEqual(breakpoints[-1], ts.sequence_length)
```

Comment:
Should be exactly equal, shouldn't they? We'd only expect numerical differences if we've performed some calculations on the values.

Reply:
Fine. I wasn't sure how the breakpoints were calculated, but if the last one is simply passed through from the seq length, then that should be fine. I'll change it.